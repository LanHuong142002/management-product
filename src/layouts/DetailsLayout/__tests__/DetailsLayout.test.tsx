import { render } from '@testing-library/react';
import { DetailsLayout } from '@layouts';

describe('Testing DetailsLayout', () => {
  it('Should renders DetailsTitle and DetailsBody correctly', () => {
    const title = <h1>Details Title</h1>;
    const body = <p>Details Body</p>;

    const { getByText, container } = render(<DetailsLayout title={title}>{body}</DetailsLayout>);

    expect(getByText('Details Title')).toBeInTheDocument();
    expect(getByText('Details Body')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
