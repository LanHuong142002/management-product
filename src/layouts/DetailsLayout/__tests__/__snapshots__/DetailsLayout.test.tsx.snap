// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing DetailsLayout Should renders DetailsTitle and DetailsBody correctly 1`] = `
<div>
  <main
    class="details-page"
  >
    <div
      class="details-title"
    >
      <h1>
        Details Title
      </h1>
    </div>
    <div
      class="details-body"
    >
      <p>
        Details Body
      </p>
    </div>
  </main>
</div>
`;
